.sublime/homebrew.sublime-project

```diff
@@ -0,0 +1,27 @@
+{
+	"folders":
+	[
+		{
+			"path": "..",
+			"folder_exclude_patterns": [
```

Comment:
How did you come up with this list? It'd be good to have a comment here and whatever location this was taken from to ensure they don't get out of sync (or, ideally, a CI test).